ygot/render.go

```diff
@@ -599,11 +600,67 @@ func KeyValueAsString(v interface{}) (string, error) {
 	}
 
 	switch kv.Kind() {
-	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
+	case reflect.Int:
+		if val, ok := v.(int); ok {
```

Comment:
Why was this done?

Reply:
This is again for performance, although it resulted in slightly more complex code. After profiling, `strconv.Itoa` and `strconv.FormatInt/FormatUint` are much more efficient than `fmt.Sprintf` so it'd be nice if we could to use them whenever possible in frequently called paths such as: `node.retrieveNodeList` (loop in recursion) -> `ygot.PathKeyFromStruct` -> `keyMapAsStrings` (loop) -> `KeyValueAsString`.

Reply:
Thanks, this looks good then. A cleaner way to write this would be to call a type switch on `v` instead of switching on each reflect.Type. This way you can avoid re-writing `return fmt.Sprintf("%d", v), nil`.

Reply:
> Thanks, this looks good then. A cleaner way to write this would be to call a type switch on `v` instead of switching on each reflect.Type. This way you can avoid re-writing `return fmt.Sprintf("%d", v), nil`.

This is a good catch, I've made the change in the latest commit. Thanks!

Reply:
Looks good, thanks!